docs/developer_manual/services/advanced/partial_results.md

```diff
@@ -0,0 +1,88 @@
+# Partial Results
+
+Producing a partial result is an optional feature that services can use to delay processing while waiting for other services to provide additional information.
+
+This additional information can come from other files within the same submission, even after the file being processed has already been otherwise completed.
```

Comment:
Should we write something about the fact that the last generated partial result will be used as the final result to the user, in case no non-partial result are generated? And that this partial result won't be used in the cache in case the file is requested to be analyzed later in another submission?